cmd/lakectl/cmd/annotate.go

```diff
@@ -29,23 +28,36 @@ var annotateCmd = &cobra.Command{
 	Args:    cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		pathURI := MustParsePathURI("path", args[0])
+		recursive, _ := cmd.Flags().GetBool("recursive")
```

Comment:
can make "recursive" const as being used for command registration (line 97) and value retrieval.

Reply:
It will "break" the way we register other flags, including "recursive" in `lakectl fs` commands. If we change it in all places it interfere with the readability of the commands regsitration, so I think it is better to leave it like that.
@Jonathan-Rosenberg - WDYT?

Reply:
Interesting... I did see that we use string literals and not variables when defining flags and retrieve their value, but is it a good practice? 
Anyway it's not a deal breaker so we can leave it that way.